enterprise/tailnet/connio.go

```diff
@@ -133,7 +133,7 @@ var errDisconnect = xerrors.New("graceful disconnect")
 
 func (c *connIO) handleRequest(req *proto.CoordinateRequest) error {
 	c.logger.Debug(c.peerCtx, "got request")
-	err := c.auth.Authorize(req)
+	err := c.auth.Authorize(c.coordCtx, req)
```

Comment:
We end up needing to pipe a few contexts around in order for Authorize to make the DB query, here and in `tailnet/peer.go` - how do we know what contexts should/shouldn't be used?

Reply:
This needs to be the `c.peerCtx` --- `coordCtx` is the context of the Coordinator's lifetime.

You should add a test that adds a subject to a context you use to call `Coordinate()` and verify the subject is there on the `Authorize` call.  Same test for the AGPL implementation.